src/moveit_studio_ur_pstop_manager/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>moveit_studio_ur_pstop_manager</name>
-  <version>4.0.1</version>
+  <version>4.0.2</version>
```

Comment:
don't include these version updates in this branch, probably from moving the base branch